### PR TITLE
actions

### PR DESCRIPTION
**Pull request by kevmoo**
- Getting ready to share
- starting
- starting by blowing up!

### CODE REVIEW

**Comment by jakemac53** [comment]
As far as I am concerned this is probably ready for a real review and publishing a pre-release.

We should add some tests before a stable release, but we have e2e testing and know it works.

**Comment by kevmoo** [comment]
> As far as I am concerned this is probably ready for a real review and publishing a pre-release.
> 
> We should add some tests before a stable release, but we have e2e testing and know it works.

Maybe @natebosch could be a 3rd set of eyes here?

**Comment by jakemac53** [comment]
Ya I can also do a review as I only wrote half of this, but it would be good for Nate to also look

File: .github/workflows/dart.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: cedx/setup-dart@v2
+        with:
+          release-channel: stable
+          version: "2.7.0"
+      - uses: actions/checkout@v2
+      - env:
+          PKGS: mono_repo
+          TRAVIS_OS_NAME: linux
```
Comment (natebosch) [inline .github/workflows/dart.yml:62]: Is the reference to travis temporary?

File: mono_repo/lib/src/mono_config.dart [hunk 2]
```diff
@@ -171,6 +176,42 @@ class MonoConfig {
 
     final mergeStages = json['merge_stages'] ?? [];
 
+    final ci = (Object ci) {
```
Comment (natebosch) [inline mono_repo/lib/src/mono_config.dart:179]: consider pulling this out into a separate method. This overall method is now 120 lines, and this initializing expression is a pretty big chunk of that.

File: mono_repo/lib/src/commands/github/generate.dart [hunk 3]
```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```
Comment (natebosch) [inline mono_repo/lib/src/commands/github/generate.dart:1]: [nit] 
```suggestion
// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
```

**Review by natebosch — APPROVED** [review]

File: .github/workflows/dart.yml (hunk 1 above)
Comment (kevmoo) [reply .github/workflows/dart.yml:62]: Yeah. It's used by the `ci` script to make sure the right thing CLI command is used

Comment (natebosch) [reply .github/workflows/dart.yml:62]: will we pick a different name in a future update?

**Pull request merged**